valuable/examples/derive.rs

```diff
@@ -1,9 +1,12 @@
+#[cfg(feature = "valuable-derive")]
```

Comment:
`valuable-derive` is not public feature, please use `derive` feature instead.

Also, instead of using cfg for all items, I would suggest moving them into the main function and using cfg only once.

Reply:
fixed, thanks for the feedback